docquality.rst

```diff
@@ -71,17 +71,12 @@ Helping with the Developer's Guide
 
 The Developer's Guide uses the same process as the main Python documentation,
 except for some small differences.  The source lives in a `separate
-repository`_.  Bug reports and patches should be submitted to the `Python
-bug tracker`_ using the ``devguide`` component.  Changes to the devguide
-are normally published within a day, on a schedule that may be different from
-the main documentation.
+repository`_ and bug reports should be submitted to the `the GitHub tracker`_.
 
-.. _separate repository: https://github.com/python/devguide
-.. _Python bug tracker: http://bugs.python.org
-
-To clone the Developer's Guide::
+To submit a :doc:`pull request <pullrequest>` you can fork the
+`devguide repo`_ to your GitHub account and clone it using::
 
-    $ git clone https://github.com/python/devguide.git
+    $ git clone https://github.com/<your_username>/devguide
```

Comment:
Is there a reason to have the ``.git`` at the end?  The ``clone`` example in the quick start doesn't have it, so I removed it from here too.